cognee/modules/retrieval/lexical_retriever.py

```
@@ -0,0 +1,121 @@
+                if fixed_document.get("type") == "DocumentChunk" and fixed_document.get("text"):
+                    try:
+                        tokens = self.tokenizer(fixed_document["text"])
+                        if not tokens:
+                            continue
+                        document_chunk = DocumentChunk.from_dict(fixed_document)
+                        new_document = get_own_properties(document_chunk)
+                        new_document["id"] = str(new_document.get("id", chunk_id))
+                        self.__class__._global_chunks[str(chunk_id)] = tokens
+                        self.__class__._global_payloads[str(chunk_id)] = new_document
+                        chunk_count += 1
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)